src/ZigZagBoomerang.jl

```diff
@@ -7,7 +7,7 @@ include("common.jl")
 include("dynamics.jl")
 export ZigZag1d, Boomerang1d, ZigZag, FactBoomerang
 const LocalZigZag = ZigZag
-export LocalZigZag, BouncyParticle, Boomerang
+export LocalZigZag, BouncyParticle, Boomerang, StickyZigZag
```

Comment:
Do you define `StickyZigZag` somewhere?

Reply:
No. It belonged to an old attempt of implementing the sticky zz